src/dev/src/app/timeline/timeline.demo.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2016-2019 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+import { Component } from '@angular/core';
+
+import { ClarityIcons } from '@clr/icons';
+
+@Component({
+  selector: 'clr-timeline-demo',
+  styleUrls: ['./timeline.demo.scss'],
+  template: `
```

Comment:
Let's remove this inline template, and use an HTML file with the `timeline-types.html` content. Then you can simplify to just have one component instead of two. See the `radios` demo directory for an example.

Reply:
Done.